generators/server/templates/src/main/java/package/service/mapper/_UserMapper.java

```diff
@@ -54,7 +53,7 @@ default User userFromId(<% if (databaseType == 'mongodb' || databaseType == 'cas
     }
 
     default Set<Authority> authoritiesFromStrings(Set<String> strings) {
-        return strings.stream().map(string -> {
+        return strings == null ? null : strings.stream().map(string -> {
```

Comment:
A null value is send from the front-end to the back-end (only the id and the login(with this PR) have values, according to the setters in UserDTO), when we save the entity that has the many-to-many relationship to the User.
And I think it should stay null because we don't do anything with the authorities at this level. We just want to save the relations (in the joint table).